### ISSUE TITLE
List file extension for non-highway travels

### ISSUE THREAD

**Issue by michihdeu** [TravelMapping/DataProcessing#605]
https://forum.travelmapping.net/index.php?topic=444.msg31809#msg31809

> Thinking ahead a bit to if/when I start accepting email submissions of .list files for things other than highways, it might be helpful to use a different extension.  It could be .rlist, .rail, or something else.

**Comment by michihdeu** [comment]
I've also opened a poll on the forum: https://forum.travelmapping.net/index.php?topic=5703

**Comment by michihdeu** [comment]
> I've also opened a poll on the forum: https://forum.travelmapping.net/index.php?topic=5703

 8 out of 10 voters are in favor of **.list, .rlist, .slist, .flist, .blist** @jteresco please go for it!

**Comment by michihdeu** [comment]
Since our railway project is getting more and more public (already some 40 users now), I think we should think about this again. Any chance to get this implemented this summer? @jteresco or @yakra ?

**Comment by jteresco** [comment]
I don't think it's too hard.  To me, it seems more important to get the railways manual settled and posted, and some of the systems activated, before accepting more user submissions.

**Comment by michihdeu** [comment]
You are right. TBH, i would not go to Production until that happened and most of the systems are reviewed.
However, I don‘t See any progress to find common rules. We will stuck like this for years. Thus, I‘d like to keep confusion for the early adaptors as low as possible from coding perspective and rename the files „soon“.

**Comment by yakra** [comment]
I admittedly need to catch up on the rail side of things.

Currently, UserData has `list_files/`, `rlist_files/`, etc.
The plan is to keep this directory structure, with subdirectories for each mode of transportation all in the main `UserData` repo, correct?
All that we'd be changing (as far as processing traveler lists is concerned) is the actual file extension to `.rlist`?

**Comment by michihdeu** [comment]
Correct. So that @jteresco sort the files submitted via email.
`list, rlist, flist` etc

**Comment by jteresco** [comment]
Yes, just so I can automate sorting of them into the right places after accumulating the day's submissions in my Downloads directory, and to avoid name conflicts there if and when someone updates lists for multiple TM instances in the same day.

**Comment by jteresco** [comment]
This change has been implemented.  Railway list files should have the `.rlist` extension from now on.  Existing files have been renamed (https://github.com/TravelMapping/UserData/commit/abbce575662450b736abbb720ec33e83eed75ae1).  I should be able to begin accepting Railway lists by email soon.

**Comment by michihdeu** [comment]
@jteresco I think that you also need to update something about data check. `/fast/tm/datacheck --rail` outputs
```
[0.4] Processing traveler list files:

[0.4] Processed 0 traveler list files.
```

Files under `/fast/home/michih/rlogs/users` are not being updated.

**Comment by jteresco** [comment]
Updated and recompiled the datacheck copy of the site update.
